models/tmp/stg_mixpanel__event_tmp.sql

```diff
@@ -0,0 +1,12 @@
+{{
+    fivetran_utils.union_data(
```

Comment:
This is dependent on the event tables having the same shape in different schemas.  How about in cases where the shape is slightly different?

Reply:
The `union_data` macro calls the [dbt_utils.union_relations](https://github.com/dbt-labs/dbt-utils?tab=readme-ov-file#union_relations-source) macro, which accounts for different colums across the two tablesi

Reply:
I tried it but for some reason it didn't work for me due to the shape of the `event` tables in different schemas. I might have done something wrong but just thought to make this known. I went with manually modelling the data myself since I had a deliverable to meet.